babel.config.js

```diff
@@ -1,9 +1,9 @@
-const defaultPresets = ['@babel/preset-react', '@babel/preset-env'];
+const defaultPresets = ['@babel/preset-react', '@babel/preset-env', '@babel/preset-flow'];
 const defaultPlugins = [
     // Adding the commonjs: true option to react-native-web plugin can cause styling conflicts
     ['react-native-web'],
 
-    '@babel/transform-runtime',
+    '@babel/plugin-transform-runtime',
```

Comment:
https://babeljs.io/docs/en/v7-migration#package-renames

Reply:
Can you elaborate on this change? Not sure I see why it's necessary.

Reply:
+1. I read through the [linked issue](https://github.com/babel/babel/issues/8379) you mention in the description, but I'm still not sure what the original issue was here that requires the Babel update.

Reply:
It's not necessary. I'll revert. I renamed it to the the whole name instead of the shorthand because on that link above it implied they (the maintainers) preferred full names for clarity to the shorthand.